app/templates/power-ups/topics.html

```diff
@@ -160,21 +188,19 @@ <h3 id="data-storage">Data Storage</h3>
   <p>Currently, there is no external API for accessing Power-Up data. Power-Up stored data can only be retrieved when the UI
     is loaded by the member with the Power-Up enabled.</p>
   <p>Members can reset their private data at any time via the Power-Ups menu.</p>
+  </hr>
 
-  <h3>Recoloring</h3>
-  <p>Trello expects you to change the color of any icons you use based on a query parameter that we will provide. This parameter
-    is designed to ensure that icons have great usability even when the background behind an image may change.</p>
-  <p>For example, card badge icons are expected to be grey (#999) in their default state, but when a color is applied to the
-    badge (such as ‘red’), the icon is passed <strong>?color=fff</strong>. This behavior means that the icon should be recolored
-    such that the primary color used is white, ensuring the image will look good and have good usability on a red background.</p>
+  <h2 id="recoloring">Recoloring</h2>
+  <p>Trello expects you to change the color of any icons you use based on a query parameter that we will provide. This parameter is designed to ensure that icons have great usability even when the background behind an image may change.</p>
+  <p>For example, card badge icons are expected to be grey (#999) in their default state, but when a color is applied to the badge (such as ‘red’), the icon is passed <strong>?color=fff</strong>. This behavior means that the icon should be recolored such that the primary color used is white, ensuring the image will look good and have good usability on a red background.</p>
+  <hr/>
 
-  <h3>Timeouts</h3>
-  <p>Your Power-Up should respond within 1 second to all capability methods. If your method takes more than 1 second to complete,
-    we will consider the callback as failed.</p>
+  <h2 id="timeouts">Timeouts</h2>
+  <p>Your Power-Up should respond within 1 second to all capability methods. If your method takes more than 1 second to complete, we will consider the callback as failed.</p>
```

Comment:
I'm going to bump this over to guidelines. I think it feels a little out-of-place here.